mysql-test/suite/innodb/t/avoid_deadlock_with_blocked.test

```diff
@@ -0,0 +1,229 @@
+--source include/have_innodb.inc
+--source include/have_debug_sync.inc
+--source include/count_sessions.inc
+
+--disable_query_log
+call mtr.add_suppression("InnoDB: Transaction was aborted due to ");
+--enable_query_log
```

Comment:
After a2d37705ca5a79149b1b064fe3a0be0bbf7eaf73 this should not be strictly necessary. But it will not hurt either.